database-plugins/src/test/java/co/cask/hydrator/plugin/db/batch/action/DBQueryActionTestRun.java

```diff
@@ -61,7 +61,10 @@ public void testAction() throws Exception {
       ImmutableMap.<String, String>builder()
         .put("connectionString", getConnectionURL())
         .put("jdbcPluginName", "hypersql")
-        .put("query", "delete from \"actionTest\" where day = '${logicalStartTime(yyyy-MM-dd,0m,UTC)}'")
+        .put("jdbcPluginType", "jdbc")
+        .put("command", "delete from \"actionTest\" where day = '1970-01-01'")
```

Comment:
don't just change property names. It breaks backwards compatibility. Just keep it as query. Also, why remove the macro?